### ISSUE TITLE
drag and drop issues

### ISSUE THREAD

**Issue by Weixuanf** [11cafe/Internal_Roadmap#10]
dragging everywhere workflow (https://github.com/chrisgoringe/cg-use-everywhere/blob/main/docs/test-workflow.json) into default workflow works, but if i drag into this SVD workflow: https://www.comfyspace.art/workflow/rRtQ4nisugx_LLtyk3xiy
It won't work, 
also someone compains RGThree not working either: https://github.com/11cafe/comfyui-workspace-manager/issues/250
please help check 😭

code pointer:
https://github.com/11cafe/comfyui-workspace-manager/blob/dev/ui/src/utils.tsx#L190-L206

basically it just uses copy paste the workflow nodes under the scene to insert them into the new workflow. If manually copy & paste workflow A nodes into workflow B works, our drag and drop should work too. But now it doesn't :(((

**Comment by arslan2012** [comment]
### For https://github.com/11cafe/comfyui-workspace-manager/issues/250 This has nothing to do with RGThree

https://github.com/chrisgoringe/cg-use-everywhere/blob/master/js/use_everywhere.js#L304-L321
![image](https://github.com/11cafe/comfyui-workspace-manager/assets/8698442/99d3fe2f-cb9a-42dd-86c7-0a70cbff7538)
use-everywhere literally hijacks and changes code of ComfyUI, so even if you are not using any "use-everywhere" nodes in the graph, it can still mess up the workflow a lot.

**Comment by arslan2012** [comment]
the main cause being, we are rendering the nodes in a temp canvas, but `use-everywhere` directly references app.canvas, there's a mismatch, causing it to get null values.

I'm thinking if there's a way for us to not use temp canvas, or we are gonna have to open a pr for `use-everywhere` to stop referncing app.canvas

**Comment by Weixuanf** [comment]
>  but use-everywhere directly references app.canvas, there's a mismatch, causing it to get null values.

~~where is this referencing? but after we copying the new nodes into the app.canvas from the temp canvas, will it still throw null?~~
nvm I think i know your meaning, it is during we create temp canvas, it will throw 

Can we temporarily let app.canvas point to our tempCanvas? Then after finish copying pointing back to original app.canvas
